### ISSUE TITLE
[A11Y] Tab navigation problems

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/polls#4782]
### Elements found:

This divider in the right sidebar traps tab navigation, not allowing users to exit:
<img width="460" height="148" alt="Image" src="https://github.com/user-attachments/assets/b5f76614-2d13-4533-8b0c-e053bec0672f" />

The combine polls screen is fully out of the tab order and cannot be used with a keyboard:
<img width="2146" height="375" alt="Image" src="https://github.com/user-attachments/assets/89d993d2-57a0-4941-920a-212ebdf7de5e" />

These checkboxes cannot be selected with tab and enter, and you cannot use arrow keys to change options. It is possible to change them using a combination of these, but it isn't intuitive.
<img width="319" height="323" alt="Image" src="https://github.com/user-attachments/assets/2d697fcc-70d6-45ec-98e6-a4116caea475" />

These checkboxes in settings cannot be changed with tab navigation
<img width="717" height="378" alt="Image" src="https://github.com/user-attachments/assets/7e7bf4ba-e29c-4a17-9342-35e9e5a3d4ec" />

These checkboxes in the voting screen are also not accessible through tab navigation
<img width="713" height="275" alt="Image" src="https://github.com/user-attachments/assets/4430fb24-9a8f-420b-9339-f70d30a0ff64" />

### ⚠️ This issue respects the following points: ⚠️

- [x] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [x] This issue is **not** already reported on Github _(I've searched it)_.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### What went wrong, what did you observe?

Some parts of the app have tab navigation "traps", and some elements are not in the tab order.

### What did you expect, how polls should behave instead?

The app should be fully navigable with tab navigation.

### What steps does it need to replay this bug?

1. Navigating the app with a keyboard, with tab, arrow, enter, etc. keys

### Affected polls version

main

### Installation method

Installed/updated from the appstore (Apps section of your site)

### Installation type

First time installation

### Can you rule out that any extension you use is involved in the issue?

- [ ] I have checked all browser extension

### Which browser did you use, when experiencing the bug?

- [x] Firefox
- [ ] Chrome
- [ ] Chromium/Chromium based (i.e. Edge)
- [ ] Safari
- [ ] Other/Don't know

### Add your browser log here

```json

```

### NC version

Nextcloud 33 (Hub 26 Winter)

### PHP engine version

PHP 8.1

### Database engine

MySQL

### Which user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other/Don't know

### Add your nextcloud server log here

```json

```

### Configuration report

```shell

```

### List of activated Apps

```shell

```

### Nextcloud Signing status

```shell

```
